oletus.mjs

```diff
@@ -30,16 +43,8 @@ export default async function test (title, implementation) {
   try {
     await implementation(assert.strict)
   } catch (e) {
-    const originalPrepareStackTrace = Error.prepareStackTrace
-    const originalStackTraceLimit = Error.stackTraceLimit
-    Error.prepareStackTrace = customPrepareStackTrace
-    Error.stackTraceLimit = 5
-    const trace = {}
-    Error.captureStackTrace(trace, test)
-    location = trace.stack || extractTraceFallback(e.stack)
```

Comment:
The problem was in the fallback case: Attempting to access `e.stack` here occasionally would return the raw Callsites, because at this point `customPrepareStackTrace` is still in effect. If that would happen, `extractTraceFallback` would throw a TypeError.

Reply:
This got me thinking - in the places where we *can* extract a custom stack from an existing Error (not sure what dictates that), we should use it. I'll add that as another commit.